include/mbedtls/cipher.h

```diff
@@ -1,12 +1,14 @@
-/**
+
+ /**
  * \file cipher.h
  *
- * \brief Generic cipher wrapper.
- *
- * \author Adriaan de Jong <dejong@fox-it.com>
- */
+ * \brief The generic cipher wrapper.
+*/
```

Comment:
Blocker: one of our sanity check scripts (`tests/scripts/check-names.sh`) chokes on `*/` at the left margin, and more generally on `*` at the left margin.

Our normal style is to indent all comment continuation lines with exactly one space, followed by `*`, followed by either one more space or by `/` if this is the end of the comment.

I personally don't care about the indentation of comment continuation lines, but we can't have this script fail. So please make sure to always start comment continuation lines with a space.

The script output for this job is https://travis-ci.org/ARMmbed/mbedtls/jobs/332787987 . I don't even know if you have access to the test results, but the output from the script is pretty hard to read in any case. So please work with us on this.

Reply:
This is what I am doing as well - this was a typo. I have gone over and fixed this - I only found one instance. Do let me know if I missed anything. I will try to make sure this doesn't happen again.